wire-library/wire-compiler/src/test/java/com/squareup/wire/schema/WireRunTest.kt

```diff
@@ -1266,7 +1266,7 @@ class WireRunTest {
         |import "squareup/options/documentation.proto";
         |
         |message Octagon {
-        |  option (documentation_url) = "https://en.wikipedia.org/wiki/Octagon";
+        |  option (options.documentation_url) = "https://en.wikipedia.org/wiki/Octagon";
```

Comment:
This example was invalid. The actual protobuf compiler, `protoc` does not allow incorrectly unqualified names like this. With these example sources, `protoc` emits the following error:

```
squareup/polygons/octagon.proto:6:10: Option "(documentation_url)" unknown. Ensure that your proto definition file imports the proto which defines the option.
```
The fix here, to include the partial package name `options`, fixes it to be valid proto source.